iron-image.html

```diff
@@ -105,9 +114,11 @@
 
   <template>
 
-    <img id="img" role="none" hidden$="[[_computeImageVisibility(sizing)]]">
-    <div id="placeholder" hidden$="[[_computePlaceholderVisibility(fade,loaded,preload)]]" class$="[[_computePlaceholderClassName(fade,loaded,preload)]]"></div>
-    <content></content>
+    <div id="sizedImgDiv" role="img" aria-label$="{{alt}}" hidden$="[[_computeBackdropHidden(sizing)]]"></div>
+    <img id="img" alt$="{{alt}}" hidden$="[[_computeImgHidden(sizing)]]">
```

Comment:
Nope - with ARIA everything needs to be explicit. With native attributes there's a bunch of implicit stuff that goes on that you just kinda ... need to memorise. :\